src/IOUService/web/ymmd.js

```diff
@@ -50,6 +77,6 @@ function show_sent_notes(notes) {
     });
 }
 
-function clear_table() {
-    $("#sent_notes_table tr").not(":first").remove();
+function clear_table(identifier) {
+    $("#" + identifier + " tr").not(":first").remove();
```

Comment:
this is ugly. combining the identifier with some other string to get the tr's and then hacking them around. Ough.

Idea: extract method: get_table_by_id(id), then use this table to clear it, and to populate it.
It's gonna be funny to decide if it's a DIV element (retrieved by getElementById, and populatable(?is this a word?) ) or jQuery ($("#..."), not populatable that easily, or something)